### DISCUSSION TITLE
How to enable PSRAM for memory allocation for Arduino_GFX library?

### DISCUSSION THREAD

**Discussion by aikboon2002** [moononournation/Arduino_GFX#214] [Q&A] [answered]
Hi!

I have PSRAM on board. 
I am wondering if I can use PSRAM for memory allocation for Arduino_GFX library?

Thanks!

**Comment by moononournation** [comment, marked as answer]
allocate for what

**Reply by aikboon2002** [reply]
Any frame buffer used in the driver in the library? or any dynamic memory allocation in the library? I want to move to PSRAM as much as possible. Thanks!

**Reply by moononournation** [reply]
if you do not use Canvas class, not much memory consumed.

**Reply by aikboon2002** [reply]
> if you do not use Canvas class, not much memory consumed.

Sounds good, thanks!
